### PR TITLE
Code cleanning in SourceFileConvertor.

### PR DESCRIPTION
**Pull request by Piezoid**
No logic changes, only superficial rewriting. I'm trying to wrap my head around
the code, anticipating a more in depth rewrite of the importing code (after deliberations).

### CODE REVIEW

File: askomics/libaskomics/SourceFileConvertor.py [hunk 1]
```diff
@@ -66,55 +81,49 @@ def get_first_lines(self, limit):
...
                                 curr_cell_type = "Entity"
-                            elif cell.isdigit() or is_float(cell):
+                            elif is_decimal(cell):
                                 curr_cell_type = "Numeric"
                             else:
                                 curr_cell_type = "Text"
                                 #OFI : We try to guess if the present colonne is a 'Category'
-                                val_cell_type[col_cpt] = []
-                                val_cell_type[col_cpt].append(cell)
+                                val_cell_type.setdefault(col_cpt, set()).add(cell)
```
Comment (abretaud) [inline askomics/libaskomics/SourceFileConvertor.py]: Hum, why are you using a set instead of list?

Comment (Piezoid) [inline askomics/libaskomics/SourceFileConvertor.py]: The purpose of set() is to count the unique values in the column, and compare this number to the number of rows. 
The old code code was building a list, allowing to count the occurrences of each values afterward (a Counter would be more appropriate), but only using the number of unique values.

Comment (abretaud) [inline askomics/libaskomics/SourceFileConvertor.py]: ok, I understand now with l115

**Comment by abretaud** [comment]
Thanks!

**Pull request merged**